src/gsIO/gsReadFile.h

```diff
@@ -44,8 +44,12 @@ class gsReadFile
 public:
 
     /** 
-        \brief Opens a file and reads an object into a pointer. This is
-        used in conjuction with a cast operator to a pointer type
+        \brief Opens a file and reads an object into a smartpointer
+        (uPtr) or an object itself. This is used in conjuction with
+        cast operators and depends on the type of object. Most types
+        are casted to uPtr. gsMultiPatch will be a gsMultiPatch object.
+        For gsBasis there exists a cast opterator to
+        std::vector<gsBasis<T>::uPtr>.
```

Comment:
What should be the result of a non working operation?

Reply:
gsReadFile should always cast to pointer, I removed the cast to multipatch and reading as object should be done as in  examples/gsPatchFromBoundary.cpp, see
https://github.com/gismo/gismo/pull/96/commits/d97380abba6ce7dcfbc2a7fc581277a6a3fa61ca